lib/virtfs/iso9660/base.rb

```diff
@@ -0,0 +1,81 @@
+require 'rufus-lru'
+require 'virtfs/nativefs/thin'
```

Comment:
No, there's no need to require that. That's the implementation of the thin interface to the native file system, not an interface definition that you need to pull in. You just need to implement the thin interface methods in your classes.